fvm/src/gas/price_list.rs

```diff
@@ -240,31 +253,29 @@ lazy_static! {
         .copied()
         .collect(),
 
-        block_memcpy_per_byte_cost: 4,
-        block_io_per_byte_cost: 2,
-        block_link_per_byte_cost: 1,
-        // TODO: PARAM_FINISH
-
-        // TODO: PARAM_FINISH
-        get_randomness_base: 1,
-        // TODO: PARAM_FINISH
-        get_randomness_per_byte: 1,
-
-        // TODO: PARAM_FINIuiSH
-        block_open_base: 1,
-        // TODO: PARAM_FINISH
-        block_read_base: 1,
-        // TODO: PARAM_FINISH
-        block_create_base: 1,
-        // TODO: PARAM_FINISH
-        block_link_base: 1,
-        // TODO: PARAM_FINISH
-        block_stat: 1,
+        get_randomness_base: 0,
+        get_randomness_per_byte: 0,
+
+        block_memcpy_per_byte_cost: 500,
+
+        block_open_base: to_milligas!(114617),
+        block_open_memret_per_byte_cost: to_milligas!(10),
+
+        block_link_base: to_milligas!(353640),
+        block_link_storage_per_byte_multiplier: to_milligas!(1),
+
+        block_create_base: 0,
+        block_create_memret_per_byte_cost: to_milligas!(10),
+
+        block_read_base: 0,
+        block_stat_base: 0,
+
+        syscall_cost: to_milligas!(14000),
+        extern_cost: to_milligas!(21000),
 
         wasm_rules: WasmGasPrices{
-            exec_instruction_cost_milli: 5000,
+            exec_instruction_cost_milli: to_milligas!(5) as u64,
```

Comment:
Gas per exec unit is defined to be 4 in FIP-0032

Reply:
Correct, this was outdated. Fixed.